src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,14 @@
+package view;
+
+public class OutputView {
+	private static final String ERROR_MESSAGE = "입력 값이 없거나, 잘못된 형식의 수식입니다.";
+
+	public void print(String result) {
+		String message = result;
+
+		if (message == null)
+			message = ERROR_MESSAGE;
```

Comment:
if문에는 대괄호를 생략하지 않는 것이 컨벤션입니다.

아래와 같이 즉시 예외를 던질 때는 사용하는 것도 괜찮습니다 :)
`if (message == null) throw IllegalStateException();`

```suggestion
		if (message == null) {
			message = ERROR_MESSAGE;
		}
```

Reply:
if문 {} 적용하였습니다.
같은 맥락에서 for, while문에도 body가 한줄이거나 아예 없는 경우에도
{}를 다 적용할지 여쭈어봅니다!!

Reply:
네 맞습니다. 다른 부분에도 모두 해당되는 사항입니다 :)